src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -217,8 +206,6 @@ public void writeToNBT(NBTTagCompound compound)
         NBTTagCompound workManagerCompound = new NBTTagCompound();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'workManagerCompound' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)